uint32 axis_requested_state
---
uint32 active_errors
uint8 axis_state
uint8 procedure_result
